.travis.yml

```diff
@@ -17,7 +17,7 @@ before_install:
   - export PATH=$M2_HOME/bin:$PATH
 script:
   - |
```

Comment:
because it makes it slow right?  
Can we document with the commits, as you go, the rationale behind the changes, so we can look back and understand a little bit?  

"why did we get rid of FOO?"
Let me check the commit log
> " Remove foo.  It is seen to cause an increase of X in Y and do z.  it is also pretty snarky and fresh"
"Oh, that makes sense"